lib/server/services/qualityControlFlag/QcFlagService.js

```diff
@@ -36,6 +36,13 @@ const { getPhysicalDplDetectorOrFail } = require('../dpl/getPhysicalDplDetectorO
  * @property {string[]} roles
  */
 
+/**
+ * Detectors which are not physcial, but which for QC flag can be created
+ */
+const VIRTUAL_DETECTORS_FOR_ASYNC_QC = {
```

Comment:
Please define the already existing `VIRTUAL` constants as `mjs` and reuse on both server and public

Reply:
done